pgbouncer-exporter/Dockerfile

```diff
@@ -14,12 +14,33 @@
 # limitations under the License.
 
 # https://hub.docker.com/r/jbub/pgbouncer_exporter
-FROM jbub/pgbouncer_exporter:v0.15.0 AS upstream
 
-FROM quay.io/astronomer/ap-base:3.18.0-2
+ARG GO_VERSION=1.21
```

Comment:
@pgvishnuram can you add a comment here indicating that this build step is a temporary workaround for CVEs and we should go back to an upstream build when whatever CVEs or golang versions changes we are waiting on have been solved upstream?